src/pydocstyle/violations.py

```diff
@@ -19,7 +22,13 @@ class Error:
     explain = False
     source = False
 
-    def __init__(self, code, short_desc, context, *parameters):
+    def __init__(
+            self,
```

Comment:
Isn't this over-indented?

Reply:
I think it should be indented once more than the code block to avoid confusion - this is PyCharm's default. Do you think otherwise?